test/pjrt/distributed_util.py

```diff
@@ -0,0 +1,88 @@
+from absl import logging
```

Comment:
why is there an empty file `test/pjrt/__init__.py`?

Reply:
That's just more formal way of making a directory as a package.